wrap/attach.go

```diff
@@ -21,7 +21,7 @@ func (c *Attach) ParseToArgs(rawArgs []string) []string {
 		// potentially harmful flags. Here this is the container to rm
 		args = append(args, "--")
 		for _, arg := range c.Cmd.Args() {
-			// Only allow attaching to named containers.
+			// If the container is given by name we prepend the user name
```

Comment:
The new comment does not reflect that `IsHexOnly` arguments are dropped as the append in line 27 is only executed for non-hex identifiers.
This currently ensures that no one can snoop around in containers created by others.

Reply:
Ok, I think this is acutally wrong in `attach` as we support hex names independent of the username for `kill`.